src/wp-includes/class-wp-scripts.php

```diff
@@ -464,7 +523,23 @@ public function print_inline_script( $handle, $position = 'after', $display = tr
 		$output = trim( implode( "\n", $output ), "\n" );
 
 		if ( $display ) {
-			printf( "<script%s id='%s-js-%s'>\n%s\n</script>\n", $this->type_attr, esc_attr( $handle ), esc_attr( $position ), $output );
+			if ( 'after-non-standalone' === $position ) {
+				printf(
+					"<script%1\$s id='%2\$s-js-after' data-wp-executes-after='%2\$s'>\n%4\$s\n</script>\n",
+					$this->type_attr,
+					esc_attr( $handle ),
+					esc_attr( $position ),
+					$output
+				);
+			} else {
+				printf(
+					"<script%1\$s id='%2\$s-js-%3\$s'>\n%4\$s\n</script>\n",
+					$this->type_attr,
+					esc_attr( $handle ),
+					esc_attr( $position ),
+					$output
+				);
+			}
```

Comment:
```suggestion
			if ( 'after-non-standalone' === $position ) {
				$output = "<script%1\$s id='%2\$s-js-after' data-wp-executes-after='%2\$s'>\n%4\$s\n</script>\n";
			} else {
				$output = "<script%1\$s id='%2\$s-js-%3\$s'>\n%4\$s\n</script>\n";
			}
			
			printf(
					$output,
					$this->type_attr,
					esc_attr( $handle ),
					esc_attr( $position ),
					$output
				);
```

Is this a little cleaner.

Reply:
Also, what about taking this opportunity to replace all manual construction of `script` tags to instead utilize `wp_print_script_tag()`?